src/ArduinoCloudThing.cpp

```diff
@@ -491,6 +528,14 @@ bool ArduinoCloudThing::ifNumericConvertToDouble(CborValue * value_iter, double
   return false;
 }
 
+void ArduinoCloudThing::resetMapDataNotBase(CborMapData * map_data) {
```

Comment:
I'd see that function as a member of class **CborMapData** as I've noted in the header file too.

Reply:
Done